source/common/stats/histogram_impl.h

```diff
@@ -68,8 +68,8 @@ class HistogramStatisticsImpl final : public HistogramStatistics, NonCopyable {
   ConstSupportedBuckets& supported_buckets_;
   std::vector<double> computed_quantiles_;
   std::vector<uint64_t> computed_buckets_;
-  uint64_t sample_count_;
-  double sample_sum_;
+  uint64_t sample_count_{0};
+  double sample_sum_{0};
```

Comment:
should we also init unit_  if we are initing the other scalars?

Reply:
It is initialized in member initializer list of both constructors, but doing it here makes it easier to see the default value.

Reply:
I don't think it can be initialized here as it is defined as `const`.

Reply:
I can move the initialization of `sample_count_` and `sample_sum_` to the MIL, if you prefer.

Reply:
I think you can add a default here even though it's const.

Reply:
TIL, thanks!
Done.